testsuite/features/support/constants.rb

```diff
@@ -967,6 +983,13 @@
         sle-manager-tools-for-micro5-pool-x86_64-5.5
         sle-manager-tools-for-micro5-updates-x86_64-5.5
       ],
+    'sle-micro-6.0' =>
+      %w[
+        sle-micro-6.0-pool-x86_64
+        sle-micro-6.0-updates-x86_64
```

Comment:
Apparently, at least for now, we don't have any updates channel.

Reply:
or manager-tools I think

Reply:
I removed the updates channel and made a note about client tools, I hope we remember to add them soon, I suspect it won't work without them.